data/src/main/java/edu/nextstep/camp/calculator/data/mapper/StatementMapper.kt

```diff
@@ -0,0 +1,16 @@
+package edu.nextstep.camp.calculator.data
```

Comment:
```suggestion
package edu.nextstep.camp.calculator.data.mapper
```
패키지명이 올바르게 되어 있지 않네요 😅